zerver/views/video_calls.py

```diff
@@ -151,3 +153,35 @@ def deauthorize_zoom_user(request: HttpRequest) -> HttpResponse:
             auth=(settings.VIDEO_ZOOM_CLIENT_ID, settings.VIDEO_ZOOM_CLIENT_SECRET),
         ).raise_for_status()
     return json_success()
+
+
+@zulip_login_required
+@never_cache
```

Comment:
I believe `never_cache` and `zulip_login_required` are not required here; they're implied by `rest_dispatch`.